### PR TITLE
Add playwright testing for the new web package

### PR DESCRIPTION
**Pull request by SergioCasCeb**
The implemented testing for the new web packages, tests only the visual aspects of the application rather than the functionality of each package. More tests will be added for the remaining console views such as OpenAPI, AsyncAPI, AAS AID, Defaults, and the visualization view.

### CODE REVIEW

**Comment by SergioCasCeb** [comment]
All remaining tests have been added as well as the respective description in the README file. Fixes #513

File: packages/web-new/tests/test.spec.js [hunk 1]
```diff
@@ -0,0 +1,2259 @@
...
+        const AASTab = page.locator("#aas-tab")
+
+        await expect(AASView).toHaveClass("console-view aas-view hidden")
+        await expect(consoleError).toHaveClass("console-view console-error hidden")
+        await expect(AASTab).toBeChecked({ checked: false })
+
+        await AASTab.click()
+
+        await expect(AASTab).toBeChecked({ checked: true })
+        await expect(AASView).toHaveClass("console-view aas-view hidden")
```
Comment (egekorkan) [inline packages/web-new/tests/test.spec.js:1605]: This one is failing with 
```
Expected string: "console-view aas-view hidden"
Received string: "console-view aas-view"
```

**Review by egekorkan — CHANGES_REQUESTED** [review]

File: packages/web-new/tests/test.spec.js (hunk 1 above)
Comment (egekorkan) [reply packages/web-new/tests/test.spec.js:1605]: I think the expected string is wrong since you click on the tab thus make it not hidden anymore

**Comment by egekorkan** [comment]
The tests are working except for one but they should not be run for all the browsers all the time and in developer machines. For example, my setup with Manjaro Linux is not supported by Playwright to install msedge. We should simply split a run with "all browsers" to another script that is run only in GitHub actions

Failing test: Open the tree visualization view with the thing template and modified the input values

**Comment by egekorkan** [comment]
@SergioCasCeb there is another test failing now where the popup events are not found so all tests for settings pop up links (check Thingweb footer link etc.) are failing. I am not sure but is that side bar opening firing a popup event by default? It seems not the case but if I add `window.dispatchEvent(new CustomEvent("popup"));` at https://github.com/eclipse-thingweb/playground/blob/master/packages/web-new/src/scripts/settings-menu.js#L60, those failing tests pass. What do you think @SergioCasCeb ?

**Comment by egekorkan** [comment]
@SergioCasCeb I have added that line of code but feel free to suggest another solution. I have also synced this branch with master.

**Comment by egekorkan** [comment]
The failure seems to be a time issue and not about the event I have mentioned above. The PR is ready to merge. The tests failing in GitHub actions is due to missing tests in web-new but these are coming with #515

**Pull request merged**